cpp/ql/lib/semmle/code/cpp/ir/rangeanalysis/internal/SsaReadPositionSpecific.qll

```diff
@@ -0,0 +1,32 @@
+private import semmle.code.cpp.ir.IR as IR
+
+class BasicBlock = IR::IRBlock;
+
+private newtype TSsaVariable = MkSsaVariable(IR::Instruction instr)
```

Comment:
This IPA doesn't seem to be used? Was `SsaVariable` supposed to extend it? (It looks like it's not actually necessary to have this type at all!)